combine/bygroup_test.go

```diff
@@ -0,0 +1,69 @@
+package combine
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestByGroupFiles(t *testing.T) {
+	Convey("Given bygroup files and an output", t, func() {
+		inputs, output, outputPath := buildByGroupFiles(t)
+
+		err := MergeByGroupFiles(inputs, output)
+		So(err, ShouldBeNil)
+
+		Convey("You can merge and compress the bygroup files to the output", func() {
```

Comment:
This convey should contain the MergeByGroupFiles() call, and the following Convey should be nested inside.